config/pmd-test.xml

```diff
@@ -198,9 +218,16 @@
   <rule ref="category/java/multithreading.xml/DoNotUseThreads">
     <properties>
       <!-- We use thread to limit the stack size for a test method. -->
+      <!-- For SuppressionFilterTest.isConnectionAvailableAndStable and
+           SuppressionsLoaderTest.loadFilterSet
+           https://github.com/checkstyle/checkstyle/issues/14870. -->
```

Comment:
Please remove `.` at the end, better to keep link without extra symbols, we have Check in CI that validate links in source code to issue tracker to be not closed.

Reply:
done